tide_site.install

```diff
@@ -479,3 +479,23 @@ function tide_site_update_8021() {
     \Drupal::service('module_installer')->install([$module]);
   }
 }
+
+/**
+ * Create new Linkit profile to view content information.
+ */
+function tide_site_update_8022() {
+  $configs = [
+    'linkit.linkit_profile.site_content' => 'linkit_profile',
+  ];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_site') . '/config/install'];
```

Comment:
You actually don't need to call `drupal_get_path()` here because you have access to `__DIR__`.  The path is always going to be relative to this `tide_site.install` file. 

```php
$config_location = [__DIR__ . '/config/install'];
```

Reply:
@christopher-hopper ok no worries - I've updated this file